dev/bots/test.dart

```diff
@@ -957,6 +957,10 @@ Future<void> _androidGradleTests(String subShard) async {
   if (subShard == 'gradle1') {
     await _runDevicelabTest('gradle_plugin_light_apk_test', env: env);
     await _runDevicelabTest('gradle_plugin_fat_apk_test', env: env);
+    await _runDevicelabTest('gradle_jetifier_test', env: env);
```

Comment:
Putting these all in `gradle1` makes that test take almost 40 minutes.  Can we split them out?

Reply:
Good point. I'd like to try parallelize these calls. Do you know how many cores each VM on  cirrus have?